monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -16,6 +16,7 @@
 from common.credentials import Credentials
 from common.types import AgentID, JSONSerializable, SocketAddress
 
+from ..i_puppet import PluginType
```

Comment:
This will result in a circular import. Why not `common/types.py?

Reply:
moved